cap/modules/theme/templates/cap_theme/header.html

```diff
@@ -48,19 +48,7 @@
         <div class="row">
             {%- block navbar_new_deposit %}
             <div class="col-xs-4">
-                {% if "deposit_groups" in session %}
-                <div class="dropdown">
-                    <a href="#" class="btn" id="header-upload" data-toggle="dropdown">
-                        <i class="fa fa-cloud-upload"></i>
-                        Create <div class="visible-lg-inline-block">An Analysis</div>
-                    </a>
-                    <ul class="dropdown-menu" aria-labelledby="header-upload">
-                        {% for group in session["deposit_groups"] %}
-                        <li><a class="dropdown-item" href="{{ url_for('cap_deposit_ui.deposit_item_new_{0}'.format(group)) }}">{{group}}</a></li>
-                        {% endfor %}
-                    </ul>
-                </div>
-                {% endif %}
+                {%- include "cap_theme/new_deposit_btn.html" %}
```

Comment:
@pamfilos I was looking at http://jinja.pocoo.org/docs/dev/extensions/#example-extension but have a look at https://github.com/sh4nks/flask-caching which should have some fixes included (fork of Flask-Cache).@pamfilos I was looking at http://jinja.pocoo.org/docs/dev/extensions/#example-extension but have a look at https://github.com/sh4nks/flask-caching which should have some fixes included (fork of Flask-Cache).